flink-yarn/src/main/java/org/apache/flink/yarn/YarnFlinkResourceManager.java

```diff
@@ -41,15 +41,20 @@
 import org.apache.hadoop.yarn.api.records.FinalApplicationStatus;
```

Comment:
Is it important to add this fix for the legacy mode?

Reply:
Not for the master and 1.7 branch but for the 1.6 branch it is necessary.